instant-xml/src/lib.rs

```diff
@@ -53,7 +69,181 @@ to_xml_for_type!(i32);
 to_xml_for_type!(String);
 
 pub trait FromXml<'xml>: Sized {
-    fn from_xml(input: &str) -> Result<Self, Error>;
+    fn from_xml(_input: &str) -> Result<Self> {
+        unimplemented!();
+    }
+
+    fn deserialize<D>(deserializer: &mut D) -> Result<Self>
```

Comment:
Should not have both `deserialize()` and `deserialize_attr()`, that should part of the `deserializer` state.

Reply:
done